packages/firestore/src/platform_browser/browser_platform.ts

```diff
@@ -47,7 +47,7 @@ export class BrowserPlatform implements Platform {
   }
 
   newConnectivityMonitor(): ConnectivityMonitor {
-    if (this.window !== undefined) {
+    if (this.window) {
```

Comment:
This could also just be `typeof window !== 'undefined'`

Reply:
this.window does exactly that, so I figured we should just use it.

Reply:
@schmidt-sebastian mind approving the PR? Thanks!